.travis.yml

```diff
@@ -65,3 +65,7 @@ script:
 
   # Code coverage
   - coverage report
+
+  # slack integration for Travis builds
+notifications:
+  slack: andela:hHgpX64Ycz7RvLVk8Z8BESUo
```

Comment:
However, I request you to use environment variables at this point as this is sensitive data that should not be exposed.

Reply:
Thanks, @reifred for pointing this out. Let me get to protecting it.